### PR TITLE
Bug 1910409: Add externalized strings for i18n for notification drawer

### PR DESCRIPTION
**Pull request by jcaianirh**
Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1910409

Externalize notification drawer strings.

### CODE REVIEW

**Review by rebeccaalpert — APPROVED** [review]
Looks good to me. I'm not an actual approver though.

File: frontend/public/components/notification-drawer.tsx
Comment (spadgett) [inline frontend/public/components/notification-drawer.tsx:90]: Don't pass `t` as a parameter. Use the global i18next instance instead.

File: frontend/packages/patternfly/src/components/notification-drawer/notification-entry.tsx
Comment (spadgett) [inline frontend/packages/patternfly/src/components/notification-drawer/notification-entry.tsx:23]: Don't pass `t` as a parameter. Use the global i18next instance instead.

Comment (spadgett) [reply frontend/packages/patternfly/src/components/notification-drawer/notification-entry.tsx:23]: No need to export if this is only used in the same file.

File: frontend/packages/patternfly/src/components/notification-drawer/notification-entry.tsx
Comment (spadgett) [inline frontend/packages/patternfly/src/components/notification-drawer/notification-entry.tsx:103]: It would be better to make the entire string one message like `Warning notification:`. This is easier for the translators to localize.

File: frontend/public/components/notification-drawer.tsx
Comment (spadgett) [inline frontend/public/components/notification-drawer.tsx:110]: Don't pass `t` as an argument

File: frontend/public/components/notification-drawer.tsx
Comment (spadgett) [inline frontend/public/components/notification-drawer.tsx:246]: Is this message actually shown in the UI? If not, we don't need to localize.

**Review by spadgett — CHANGES_REQUESTED** [review]

File: frontend/public/components/notification-drawer.tsx
Comment (jcaianirh) [reply frontend/public/components/notification-drawer.tsx:246]: @spadgett this is returned as loadError from the UI store which is displayed in the component AlertErrorState when an error occurs and no alerts are loaded.

**Comment by jcaianirh** [comment]
@spadgett made requested changes thanks for the review

File: frontend/packages/patternfly/src/components/notification-drawer/notification-entry.tsx
Comment (spadgett) [inline frontend/packages/patternfly/src/components/notification-drawer/notification-entry.tsx:22]: You should declare this method inside the component since it's only used in one place. That way you can use the `t` from `useTranslation` instead of the global `i18next` instance.

Comment (jcaianirh) [reply frontend/packages/patternfly/src/components/notification-drawer/notification-entry.tsx:22]: sounds good @spadgett

**Review by spadgett — APPROVED** [review]
lgtm, please squash

**Comment by jcaianirh** [comment]
@spadget squashed

/hold for approvals

/assign @ahardin-rh @yapei @sferich888

**Comment by yapei** [comment]
spin up a cluster and tested notification drawer functions, such as
Screen Shot 2021-01-11 at 2 32 45 PM

no issues found

**Comment by spadgett** [comment]
/unassign @sferich888 
/assign @reestr 

Strictly speaking, this shouldn't need approvals as it's a bug fix. I don't mind getting sign off, though, and it's good that QE has tested.

**Comment by jcaianirh** [comment]
@spadgett I think this needs approvals only because this is part of the i18n story, but never got addressed, so its external to the jira story as a bug fix, so I think it should go through the same approval process.

**Comment by spadgett** [comment]
Let's go ahead and remove the hold as this is a bug fix.

/hold cancel

**Comment by jcaianirh** [comment]
@spadgett could you add an lgtm on this one

**Review by spadgett — APPROVED** [review]
/lgtm

**Comment by spadget** [comment]
Please can you remove me from this thread?

I was only included in the first place because someone wrote the wrong
username.

**Comment by spadgett** [comment]
> Please can you remove me from this thread? I was only included in the first place because someone wrote the wrong username.

I have no way to remove you (sorry), but GitHub has an Unsubscribe button you can use.

**Comment by spadgett** [comment]
@jcaianirh Needs rebase :/

**Comment by jcaianirh** [comment]
@spadgett rebased, thanks

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**
